doc/connectivity/networking/api/wifi.rst

```diff
@@ -28,3 +28,40 @@ API Reference
 *************
 
 .. doxygengroup:: wifi_mgmt
+
+X.509 Certificate header generation
```

Comment:
Looking at https://builds.zephyrproject.io/zephyr/pr/74847/docs/connectivity/networking/api/wifi.html

this should be before the doxygen docs i.e., before `API reference`